app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php

```diff
@@ -55,16 +70,16 @@ public function __construct(
      * Validates that the email address isn't being used by a different account.
      *
      * @param string $email
-     * @throws \Magento\Framework\Exception\LocalizedException
+     * @throws LocalizedException
      * @return void
      */
-    protected function validateEmailAvailable($email)
+    protected function validateEmailAvailable(string $email): void
```

Comment:
Please revert adding types of arguments and return types for **all methods that were existing there**. It's signature change, that is not backward compatible